gen/GNUmakefile

```diff
@@ -6,6 +6,7 @@ CFLAGS+=	-I.. -I/usr/local/include -g -DHTDOCS=\"${PREFIX}/share/ipgen/htdocs\"
 CFLAGS+=	-Wall -Wstrict-prototypes -Wmissing-prototypes -Wpointer-arith
 CFLAGS+=	-Wreturn-type -Wswitch # -Wshadow XXX for gen.c
 CFLAGS+=	-Wcast-qual -Wwrite-strings
+CFLAGS+=	-Wextra -Wno-unused-parameter
```

Comment:
Let's keep warnings or fix them instead of hiding them by `-Wno-unused-parameter` that would hide future issues.